graph/src/loader.c

```diff
@@ -39,6 +39,14 @@ void rage_element_loader_free(rage_ElementLoader * el) {
     free(el);
 }
 
+static char * join_path(char * a, char * b) {
+    char * joined = malloc(strlen(a) + strlen(b) + 2);
```

Comment:
Why is this plus 2? Does `strlen` omit the null byte on the end, but you have to account for it when you allocate, along with the extra `'/'` you're adding?

Reply:
Yup